.circleci/config.yml

```diff
@@ -100,7 +100,6 @@ commands:
             BILLING_PROJECT: << parameters.billing_project >>
           command: |
             mkdir -p ${SCREENSHOT_DIR}
-            TERRA_TOKEN=$(node scripts/getBearerToken.js "${ALPHA_TEST_UA}" "${ALPHA_USER_SA_KEY_JSON}") \
```

Comment:
@panentheos I was able to remove this line, as you suggested, by adding the service account key to the circle environment with the right name.